data/org.gnome.shell.gschema.xml.in

```diff
@@ -103,6 +103,16 @@
         for product developers.
       </description>
     </key>
+    <key name="enable-behind-the-screen" type="b">
```

Comment:
I wonder if we should rename that to "enable-code-view" as long as we have not deployed that feature (in an enabled by default) widely.Update frustrations should still be negotiable.